### ISSUE TITLE
Allow for specific linux builds.

### ISSUE THREAD

**Issue by jostrander** [MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-#894]
The build tooling should allow a package to be built for any one specific distribution so specific packages can be built without building for both architectures. Thoughts? This would specifically help the Arch community out so they can clone, build and update via AUR without needing both architecture libraries available.

**Comment by BrianAllred** [comment]
As stated over on issue [878](https://github.com/MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-/issues/878), building both architectures by default limits those who want to build from source to 64-bit distributions with multiarch/multilib enabled only. Not limited to just Arch, either.

**Comment by MarshallOfSound** [comment]
Hmmm, interesting.  This was always a design decision based of the assumption that we would never be building one without the other (I was not aware of AUR at that point).

I guess I'll have to figure out how to split them up without parrerellizing them :thought_balloon:

**Comment by jostrander** [comment]
Well at any rate, if someone were to clone this on a 32 bit system and try building they would have to also sit through the 64 bit build as it failed horribly, so separating it out would benefit in that sense.

**Comment by MidnightJava** [comment]
Can the rpm be installed on CentOS 6.7? I have 2.6.32-573.22.1.el6.x86_64 and it said "wrong architecture" when I tried to install the 64-bit Fedora RPM. So I installed the 32-bit Fedora rpm successfully. But after installing 32-bit dependencies one at a time based on "missing library" messages when trying to run the google-music player, I reached the error

google-play-music-desktop-player: /usr/lib/libstdc++.so.6: version `GLIBCXX_3.4.15' not found (required by /usr/share/google-play-music-desktop-player/libnode.so)

And there doesn't seem to be any way to satisfy that. I have the latest versions of libstdc++ and glibc installed.
